retrieval/scrape.go

```diff
@@ -74,12 +74,29 @@ var (
 		},
 		[]string{interval},
 	)
+	targetSyncIntervalLength = prometheus.NewSummary(
```

Comment:
There's one scrape pool per job. Having the a `scrape_job` dimension with that would be helpful.